### ISSUE TITLE
dark mode 

### ISSUE THREAD

**Issue by kadarisman** [ixartz/Next-JS-Landing-Page-Starter-Template#29]
dark and light mode is not available yet?
what if I am interested in developing it? is it allowed

**Comment by ixartz** [comment]
Currently, there is no dark/light mode available yet.
Of course, it's allowed and you can also contribute it. You can definitively open a pull request. Thank you so much.

**Comment by maneeshkumarr** [comment]
i want to work on this issue

**Comment by maneeshkumarr** [comment]
Hi! I see that this issue was mentioned a while ago, but it seems like it's still open. I’d love to take a shot at fixing the dark mode issue if that's okay. Please let me know if you’re still planning on working on it or if I should go ahead with the fix.

**Comment by ixartz** [comment]
Hi, thank you for your comment, please go ahead. Totally open to PR.

But, it might be better to update to project to the latest version like Next.js 15 and update to Tailwind CSS 4.
Then, we can add the dark mode, what do you think?

**Comment by maneeshkumarr** [comment]
Yes, I agree. Updating to Next.js 15 and Tailwind CSS 4 sounds like a good idea to stay up-to-date with the latest features and improvements.

**Comment by maneeshkumarr** [comment]
Just to clarify, I'm working with the current versions of Tailwind CSS and Next.js used in the project — I'm not upgrading them. This is to avoid introducing compatibility issues or breaking changes while focusing on implementing the dark mode feature smoothly.

**Comment by maneeshkumarr** [comment]
But in dark mode, if some content (like text or icons) remains in dark colors and becomes hard to see — should I also handle updating those styles for better readability?
